docs/framework.md

```diff
@@ -1458,55 +1276,39 @@ P   U
 
 </div>
 
-Information on the eligibility criteria for bidders can be provided using `tender` section.
-
-**Schema**: Information can be provided using the following OCDS fields.
-
-```eval_rst
-.. jsonschema:: _static/patched/release-schema.json
-    :include: tender/eligibilityCriteria
-    :collapse:
-    :nocrossref:
-```
-
-**JSON example:**
+Use [Document](reference/documents) objects in the [tender/documents](../reference/schema/#release-schema.json,,tender/documents) array to provide links to procurement documents, feasibility studies, including land acquisition, social, environmental, and rehabilitation related information and reports of independent procurement auditors. A short summary of each document can be provided using its `description` field. Each document's `documentType` field should be set to a value from the [document type codelist](../reference/codelists/#documenttype), to identify the type of document being disclosed.
```

Comment:
The "including land acquisition, social, environmental, and rehabilitation related information" in the middle of a list of document types reads a bit odd.

Reply:
Yes, it's based on this part of the framework which also reads oddly:

> links to all procurement documents, final feasibility study(including cost-benefit analysis, if any), land acquisition, rehabilitation, social (including information on human rights impact assessment) and environmental assessment, related information, and reports of independent procurement auditors (if any).

How about:

```markdown
Use [Document](reference/documents) objects in the [tender/documents](../reference/schema/#release-schema.json,,tender/documents) array to provide links to documents.

You should provide links to:

* all procurement documents 
* feasibility studies, including cost-benefit analyses, if available
* land acquisition, rehabilitation, social and environmental assessments, including information on human rights impact assessments
* reports of independent procurement auditors

A short summary of each document can be provided using its `description` field. Each document's `documentType` field should be set to a value from the [document type codelist](../reference/codelists/#documenttype), to identify the type of document being disclosed.
```

Reply:
That's much clearer. I think we can change one bullet to:

```
* land acquisition, rehabilitation, social, human rights, and environmental assessments
```

The original wording I guess treats human rights assessments as a sub-type of social assessment, but I don't think this concept hierarchy is important to retain.

Reply:
Updated in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/ef0651de03805ab5f86c3493e16fb4452b0422ae